src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-types/lancie-admin-ticket-options.html

```diff
@@ -0,0 +1,164 @@
+      </div>
+      <div class="card-actions">
+        <paper-button on-tap="openAddOptionDialog">Add Ticket Option</paper-button>
+      </div> 
+    </paper-card>
+    <lancie-dialog id="addTicketOptionDialog">
+      <h2>Add Ticket Option</h2>
+      <div class="dialog-content">
+        <lancie-error id="addDialogError"></lancie-error>
+        <lancie-form id="addTicketOptionForm" refurl="tickets/options" on-response="onAddOptionResponse">
+          <paper-input label="Name" name="name" auto-validate required error-message="This field is required!"></paper-input>
+          <paper-input label="Price" name="price" step=".01" type="number" auto-validate required error-message="This field is required!"></paper-input>
+        </lancie-form>
+      </div>
+      <div class="dialog-actions">
+        <paper-button on-tap="tryAddOption">Add</paper-button>
+      </div>
+    </lancie-dialog>
+
+    <lancie-dialog id="deleteTicketOptionDialog">
+      <h2>Confirm deletion</h2>
+      <div class="dialog-content">
+        <lancie-error id="dialogError"></lancie-error>
+        <h4>Are you certain you want to delete this ticket option?</h4>
+        <p>[[deleteOption.name]]</p>
+      </div>
+      <div class="dialog-actions">
+        <paper-button dialog-dismiss>Cancel</paper-button>
+        <paper-button on-tap="tryOptionDelete" class="confirm-delete-button">Delete</paper-button>
+      </div>
+    </lancie-dialog>
+  </template>
+  <script>
+    (function () {
+      'use strict';
+
+      Polymer({
+        is: 'lancie-admin-ticket-options',
+        properties: {
+          data: Object,
+          mappings: {
+            type: Object,
+            value: function() {
+              return {
+              };
+            }
+          },
+          renderers: {
+            type: Object,
+            value: function() {
+              return {
+                possibleOptions: function(data) {
+                  var text = data.reduce(function(text, t){
+                    return text + t.name + ' ' + t.price +', ';
+                  }, '');
+                  return text.substr(0, text.length - 2);
+                },
+              };
+            }
+          }
+        },
+
+        tryAddOption: function(e, request) {
+          this.$.addTicketOptionForm.validateAndSubmit();
+          this.$.addTicketOptionForm.reset();
+        },
+
+        openAddOptionDialog: function() {
+          this.$.addTicketOptionForm.reset();
+          this.$.addTicketOptionDialog.open();
+        },
+
+        onAddOptionResponse: function(e, request) {
+          if (request.succeeded) {
+            this.$.addTicketOptionDialog.close();
+            this.$.getOptionAjax.generateRequest();
+          } else {
+            this.$.addDialogError.setError("Could not add option. Please refresh and try again.")
+          }
+        },
+
+        deleteTicketOptionDialog: function(event) {
+          this.deleteOption = this.options.filter(option => {
```

Comment:
What you're looking for here is a `.find()`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find